examples/genes_analysis.Rmd

```diff
@@ -0,0 +1,85 @@
+---
+title: "Analyzing genes of interest with SolveBio"
+output: rmarkdown::html_document
+---
+```{r, message = FALSE, warning=FALSE}
+rm(list=ls())
+library(solvebio)
+library(data.table)
+```
+login to SolveBio:
+```{r eval=FALSE}
+login(api_key="<api_key>", api_host="<api_host>")
+```
+Define a list of genes for the analysis and create a filter that will be used to query SolveBio datasets:
```

Comment:
could you add links somewhere to the data on SolveBio? like a comment or something with a link to my.solvebio.com/data/clinvar, etc